test/functional/repositories_filesystem_controller_test.rb

```diff
@@ -45,17 +43,20 @@ def setup
     @request    = ActionController::TestRequest.new
     @response   = ActionController::TestResponse.new
     User.current = nil
-    Setting.enabled_scm = Setting.enabled_scm.dup << 'Filesystem' unless Setting.enabled_scm.include?('Filesystem')
-    @repository = Repository::Filesystem.create(
-                      :project => Project.find(PRJ_ID),
-                      :url     => REPOSITORY_PATH,
-                      :path_encoding => nil
-                      )
+
+    with_existing_filesystem_scm do |repo_path|
+      @repository = Repository::Filesystem.create(
+          :project => Project.find(PRJ_ID),
```

Comment:
Indent the first parameter one step more than the previous line.<br>Use the new Ruby 1.9 hash syntax.